src/components/views/messages/MPollBody.tsx

```diff
@@ -182,12 +182,14 @@ export default class MPollBody extends React.Component<IBodyProps, IState> {
     private addListeners(): void {
         this.state.poll?.on(PollEvent.Responses, this.onResponsesChange);
         this.state.poll?.on(PollEvent.End, this.onRelationsChange);
+        this.state.poll?.on(PollEvent.UndecryptableRelations, this.render.bind(this));
```

Comment:
Instead of calling render, would it be better to add the undecryptable event count to the state?

Reply:
I agree that this is kind of weird, but I also can't think of a concrete reason that duplicating the event count in MPollBody state would be better.